src/components/ComingSoon.js

```diff
@@ -0,0 +1,212 @@
+    filter: 'brightness(85%)',
+    margin: '0 auto',
+    display: 'flex',
+    minHeight: 'calc(100vh - 64px)',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    color: '#fff',
+    [theme.breakpoints.down('sm')]: {
+      paddingTop: theme.spacing(2),
+      paddingBottom: theme.spacing(2),
+    },
+  },
+  comingSoon: {
+    marginBottom: theme.spacing(2),
+  },
+  countdownPaper: {
+    borderRadius: '50%',
+    height: '150px',
+    width: '150px',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    border: `1px solid ${theme.palette.secondary.main}`,
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(2),
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(0),
+      marginRight: theme.spacing(0),
+      marginTop: theme.spacing(1),
+      marginBottom: theme.spacing(1),
+    },
+  },
+  countdownFont: {
+    color: theme.palette.secondary.main,
+  },
+  countdownContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+    },
+  },
+}))
+const ComingSoon = ({
+  pageContext: {
+    data: {
+      metadata: {
+        url,
+        title,
+        ogTitle,
+        ogDescription,
+        ogImage,
+        causeSpecificKeywords,
+      },
+      sections: {
+        comingSoon: { name, date },
+      },
+    },
+  },
+  location,
+}) => {
+  const cx = useStyles()
+  const absolutePageURL = getAbsoluteURL(location.pathname || '')
+  const ogImgURLAbsolute = getAbsoluteURL(
+    get(ogImage, 'childImageSharp.gatsbyImageData.images.sources[0].srcSet', '')
+  )
+  const canonicalURL = getAbsoluteURL(url)
+  return (
+    <div>
+      <HeadTags
+        title={title}
+        titleTemplate="Tab for A Cause"
+        ogTitle={ogTitle}
+        ogDescription={ogDescription}
+        ogImage={ogImgURLAbsolute}
+        keywords={KEY_WORDS.concat(causeSpecificKeywords)}
+        pageURL={absolutePageURL}
+      />
+      <Helmet>
+        <link rel="canonical" href={canonicalURL} />
+      </Helmet>
+      <AppBar color="primary" position="sticky">
+        <Toolbar>
+          <div className={cx.logoContainer}>
+            <div
+              data-test-id="logo-container"
+              style={{
+                display: 'flex',
+                justifyContent: 'flex-start',
+                alignItems: 'center',
+              }}
+            >
+              <Link to={homeURL}>
+                <img
+                  data-test-id="tab-logo-with-text"
+                  src={logoWhite}
+                  style={{ height: 40 }}
+                />
+              </Link>
+            </div>
+          </div>
+
+          <MoneyRaisedDisplay whiteClassName={cx.whiteFont} />
+        </Toolbar>
+      </AppBar>
+      <div className={cx.titleSection}>
+        <Typography variant="h2" color="inherit">
+          TAB FOR {name}
```

Comment:
Use CSS text-transform to capitalize so the styling doesn't rely on the input name.